openapi-style-guide.md

```diff
@@ -25,17 +25,24 @@
   * [Request body](#request-body)
   * [Response body](#response-body)
   * [Support for pagination](#support-for-pagination)
+  * [Long-running operations](#long-running-operations)
```

Comment:
I'm guessing all these added headings in the TOC were just missing before?

Maybe instead of keeping them up to date, we direct people to use the GitHub-generated TOC:

<img width="360" alt="image" src="https://user-images.githubusercontent.com/1532486/175649404-9f5da8f4-3861-4a37-93df-dd7df2463827.png">